settings.js

```diff
@@ -284,30 +282,30 @@ function generate(){
 
   // street suffix filters (replace text inside tokens)
   // based off synonym list
-  street_suffix.synonyms.forEach( function( synonym ){
+  synonyms.street_suffix_contractions.forEach( function( synonym ){
     var split = synonym.split(' ');
     settings.analysis.filter[ "keyword_street_suffix_" + split[0] ] = {
       "type": "pattern_replace",
       "pattern": " " + split[0],
       "replacement": " " + split[2]
-    }
+    };
   });
 
   // compass prefix filters (replace text inside tokens)
-  // based off direction_synonyms list
-  street_suffix.direction_synonyms.forEach( function( synonym ){
+  // based off directionals list
+  synonyms.directionals.forEach( function( synonym ){
     var split = synonym.split(' ');
     settings.analysis.filter[ "keyword_compass_" + split[0] ] = {
       "type": "pattern_replace",
       "pattern": split[0],
       "replacement": split[2]
-    }
+    };
   });
 
   // Merge settings from pelias/config
-  if( 'object' == typeof config &&
-      'object' == typeof config.elasticsearch &&
-      'object' == typeof config.elasticsearch.settings ){
+  if( 'object' === typeof config &&
+      'object' === typeof config.elasticsearch &&
+      'object' === typeof config.elasticsearch.settings ){
```

Comment:
please tell me this wasn't causing some subtle issue :P

Reply:
doubt it, just got caught by the linter :)